### PR TITLE
databases-week1

### PR DESCRIPTION
**Pull request by veronicaburduzhan**
Thanks in advance for your comments and review:)

### CODE REVIEW

File: databases/week1/class/homework.sql [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT 
+  * 
+FROM 
+  task 
+  JOIN status ON task.status_id = status.id 
+WHERE 
+  status.name LIKE "%done%";
```
Comment (lawansubba) [inline databases/week1/class/homework.sql]: status.name = 'done';
_Pointer_
Use LIKE operator for only pattern matches

File: databases/week1/class/homework.sql [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+-- Find all the tasks that are not marked as done
+SELECT 
+  * 
+FROM 
+  task 
+  JOIN status ON task.status_id = status.id 
+WHERE 
+  NOT status.name LIKE "%done%";
```
Comment (lawansubba) [inline databases/week1/class/homework.sql]: status.name != 'done';
Pointer
Use LIKE operator for only pattern matches

File: databases/week1/class/homework.sql [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  task 
+ORDER BY 
+  created DESC;
+
+-- Get the single most recently created task
+SELECT 
+  MAX(created) 
+FROM 
+  task;
+
```
Comment (lawansubba) [inline databases/week1/class/homework.sql:48]: SELECT
*
FROM
task
ORDER BY created DESC
LIMIT 1;

_Pointer_
Objective is to find the task that was the most recently created.

File: databases/week1/class/homework.sql [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT 
+  title, 
+  due_date 
+FROM 
+  task 
+WHERE 
+  title LIKE '%database%';
```
Comment (lawansubba) [inline databases/week1/class/homework.sql]: title LIKE '%database%' OR description LIKE '%database%'
_Pointer_
Missing the second part of the task

File: databases/week1/class/homework.sql [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+SELECT 
+  task.title, 
+  status.name 
+FROM 
+  task 
+  INNER JOIN status ON task.status_id = status.id;
+
+-- Get the name of each status, along with a count of how many tasks have that status
+SELECT 
+  COUNT(task.title),
```
Comment (lawansubba) [inline databases/week1/class/homework.sql]: COUNT(task.title) AS COUNT
_Pointer_
For columns that you create provide a column name

**Review by lawansubba — COMMENTED** [review]
Good work!! Please address the few comments  :)

File: databases/week1/class/homework.sql (hunk 1 above)
Comment (veronicaburduzhan) [reply databases/week1/class/homework.sql]: Thanks, now it's clear for me! I wasn't sure about it 'cause equality operator didn't work in some cases (now, I assume this is as you've mentioned "pattern cases").

**Pull request merged**